Source/WebCore/page/ResizeObserver.cpp

```diff
@@ -162,9 +168,14 @@ void ResizeObserver::deliverObservations()
         return ResizeObserverEntry::create(observation->target(), observation->computeContentRect(), observation->borderBoxSize(), observation->contentBoxSize());
     });
     m_activeObservations.clear();
-    auto activeObservationTargets = std::exchange(m_activeObservationTargets, { });
 
-    auto targetsWaitingForFirstObservation = std::exchange(m_targetsWaitingForFirstObservation, { });
+    Vector<WeakPtr<Element, WeakPtrImplWithEventTargetData>> activeObservationTargets;
```

Comment:
I suspect this part of the PR was the cause of rdar://164271295. Now that those vectors on the stack no longer contains GCReachableRef types, only WeakPtr, what's keeping the targets alive? Those stack vectors are unused, they were only there to keep the targets alive. But now they contain WeakPtrs so they are unused and do nothing.